quickstarts/gcp/gcp-alloydb/config.yml

```diff
@@ -0,0 +1,34 @@
+slug: gcp-alloydb
+description: |-
+  ## What is AlloyDB?
+
+  A fully managed PostgreSQL-compatible database service for your most demanding enterprise database workloads. AlloyDB combines the best of Google 
+  with one of the most popular open-source database engines, PostgreSQL, for superior performance, scale, and availability.
+
+  ### Get started!
+
+  Start monitoring AlloyDB by connecting Google Cloud Platform (GCP) to New Relic!
+  Check out our AlloyDB documentation to instrument your cloud service and manage the stability, scalability, and reliability of your systems with 
+  New Relic's infrastructure monitoring capabilities.
+
+  ### More info
+
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-alloydb-monitoring-integration/) to learn more about New Relic monitoring for AlloyDB.
+summary: Monitor AlloyDB by connecting GCP to New Relic
+icon: icon.png
+level: New Relic
+authors:
+  - New Relic
+title: AlloyDB
+documentation:
+  - name: AlloyDB integration docs
+    description: Monitor AlloyDB by connecting GCP to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-alloydb-monitoring-integration/
+keywords:
+  - gcp
+  - google cloud platform
+  - alloydb
+installPlans:
```

Comment:
Please reference a data source instead. The data source equivalent for this install plans is: google-cloud-platform

Reply:
@brammerl do you mean replace `installPlans:` with dataSource:` ?

Reply:
yes.

Reply:
done